config/makeprops.py

```diff
@@ -582,19 +582,19 @@ def fix(self, propertyName):
 
     def deprecatedImpl(self, propertyName):
         self.srcFile.write(
-            '             new Property(@"^%s\.%s$", true, null),\n'
+            r'             new Property(@"^%s\.%s$", true, null),\n'
```

Comment:
If I understand this correctly, these are escape sequences for the regexp, in a python string it should be `\\.` or use the `r""` format which doesn't use escape sequences. Is that correct?

Reply:
Yes, other similar fixes in the past used the r"" format.